netlify.toml

```diff
@@ -1,6 +1,10 @@
 # netlify configuration
 [build]
-publish = "site"
-command = "make build-docs-netlify"
+publish = "."
```

Comment:
Instead of building off updates to main and running the old make command, now Netlify:
* builds of `docs` branch
* publishes the content on the docs branch for all the versioned folders, not just the `site` folder
* doesn't need to run a build command, because `mike` already builds the content to HTML files in the `docs` branch